aim/sdk/uri_service.py

```diff
@@ -52,7 +53,10 @@ def request_batch(self, uri_batch: List[str]) -> Iterator[Dict[str, bytes]]:
 
             # TODO: [MV] change to some other implementation of view when available
             #  which won't collect in case of custom objects
-            yield {uri: container.tree().view(resource_path).collect()}
+            data = container.tree().subtree(resource_path).collect()
+            if isinstance(data, BLOB):
+                data = data.load()
+            yield {uri: data}
 
         # clear container pool
         self.container_persistent_pool.clear()
```

Comment:
If there are two parallel `request_batch()` calls, the moment first finishes, it clears the pool of the second as well.

Reply:
yeah, that will need an improvement in the future, but as of now, URI service is used by the UI only, so in that case we'll be fine, as each request creates it's own service.